src/NSwag.SwaggerGeneration.AspNetCore/Processors/OperationResponseProcessor.cs

```diff
@@ -75,10 +75,11 @@ public OperationResponseProcessor(AspNetCoreToSwaggerGeneratorSettings settings)
                     var returnType = apiResponse.Type;
                     var response = new SwaggerResponse();
                     string httpStatusCode;
-                    if (apiResponse.StatusCode == 0 && IsVoidResponse(returnType))
-                        httpStatusCode = "200";
-                    else if (apiResponse.TryGetPropertyValue<bool>("IsDefaultResponse"))
+
+                    if (apiResponse.TryGetPropertyValue<bool>("IsDefaultResponse"))
```

Comment:
`IsDefaultResponse` needs to be prioritized over the "0" status code. Added a couple of unit tests to cover the behavior of `ProcessAsync`

Reply:
Thanks for the PR. Can you delete UpgradeLog2.htm? Maybe you should work in NSwag.Min.sln, this one is less heavy...

Reply:
Everything looks good, thank you!

Reply:
Done! I think it did not have the AspNetCore.Tests project so I had to open the other one.